configure.ac

```diff
@@ -496,6 +496,7 @@ case $host in
      AC_CHECK_LIB([shlwapi],      [main],, AC_MSG_ERROR(libshlwapi missing))
      AC_CHECK_LIB([iphlpapi],      [main],, AC_MSG_ERROR(libiphlpapi missing))
      AC_CHECK_LIB([crypt32],      [main],, AC_MSG_ERROR(libcrypt32 missing))
+     AC_CHECK_LIB([dwmapi],      [main],, AC_MSG_ERROR(libdwmapi missing))
```

Comment:
Do you know what is the first release of windows to have `dwmapi`? Is this a recent addition? (if it's windows 7 or older it's fine)

Reply:
https://docs.microsoft.com/en-us/windows/win32/api/dwmapi/nf-dwmapi-dwmsetwindowattribute suggests at least Vista has it.